doc/source/notebooks/intro_to_gpflow2.pct.py

```diff
@@ -361,4 +364,47 @@ def checkpointing_training_loop(model: gpflow.models.SVGP,
 
 # %%
 params = gpflow.utilities.parameter_dict(model)
-gpflow.utilities.multiple_assign(model, params)
+gpflow.utilities.multiple_assign(model, params)\
+
+# %% [markdown]
+# ### TensorFlow saved_model
+#
+# At present tensorflow does not support saving custom variables like instances of the `gpflow.base.Parameter` class, see [here](https://github.com/tensorflow/tensorflow/issues/34908).
+#
+# However, once trained completed, it is possible to clone the model and replace all Parameters with constants holding the same value:
+
+# %%
+model
+
+# %%
+frozen_model = gpflow.utilities.freeze(model)
+
+# %% [markdown]
+# In order to save the model we need to define a `tf.Module` holding the `tf.function`'s that we wish to export, as well as a reference to the underlying model:
+
+# %%
+module_to_save = tf.Module()
+predict_fn = tf.function(frozen_model.predict_f, input_signature=[tf.TensorSpec(shape=[None, 1], dtype=tf.float64)], autograph=False)
```

Comment:
does this approach allow to store multiple functions?

Reply:
You store functions that you want to store. You have to assign them as an attribute. The explanation will come in the following issue (we discussed it offline).

Reply:
The answer is "yes you can store multiple functions".